roles/neutron-common/templates/etc/neutron/policy.json

```diff
@@ -88,6 +101,7 @@
     "update_router:external_gateway_info:enable_snat": "rule:admin_only",
     "update_router:distributed": "rule:admin_only",
     "update_router:ha": "rule:admin_only",
+    "_COMMENT": "required for horizon",
```

Comment:
The following lines are what horizon checks to show certain buttons and allow certain actions. For whatever reason, they're separate checks than those used for the CLI.